src/pyseekdb/client/client_seekdb_server.py

```diff
@@ -114,6 +114,65 @@ def get_raw_connection(self) -> pymysql.Connection:
         """Get raw connection object"""
         return self._ensure_connection()
     
+    def detect_db_type_and_version(self) -> Tuple[str, str]:
```

Comment:
This function works both for seekdb-embedded, seekdb-server and oceanbase.
So, we should move the function to a more proper place?